homeassistant/components/verisure/__init__.py

```diff
@@ -68,6 +70,10 @@
 
 CAPTURE_IMAGE_SCHEMA = vol.Schema({vol.Required(ATTR_DEVICE_SERIAL): cv.string})
 
+DISABLE_AUTOLOCK_SCHEMA = vol.Schema({vol.Required(ATTR_DEVICE_SERIAL): cv.string})
```

Comment:
Why do you need a serial ?

Reply:
To be able to enable or disable autolock for a certain lock. device_label would probably be a better name than device_serial. But since it's called serial for the camera capture service, i'll continue with that name.